crates/goose/src/config/extensions.rs

```diff
@@ -134,3 +134,24 @@ pub fn get_enabled_extensions() -> Vec<ExtensionConfig> {
         .map(|ext| ext.config)
         .collect()
 }
+
+pub fn get_warnings() -> Vec<String> {
```

Comment:
I chose to not unit test this because it is more distracting in this case, and hopefully we can reap all sse code in a version or two including this